src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-details/osd-details.component.html

```diff
@@ -1,19 +1,25 @@
 <tabset *ngIf="selection"
-        id="tabset-osd-details">
-  <tab heading="Devices"
+        stateId="tabset-osd-details">
```

Comment:
This is breaking the E2E tests: https://dashboard.cypress.io/projects/k7ab29/runs/45/failures/618f00d3-c243-4f93-b8ac-fce2c32bfff8
Either restore the id or change the test.